tensorrt_llm/_torch/pyexecutor/_util.py

```diff
@@ -210,6 +210,7 @@ def estimate_max_kv_cache_tokens(py_executor: PyExecutor,
     py_executor.start_worker()
     py_executor.await_responses(req_ids)
     # TODO check why call mpi_barrier() here will hang-on, but call mpi_allgather(0) is fine.
+    mpi_allgather(0)
```

Comment:
Could you pleae unwaive `accuracy/test_llm_api_pytorch.py::TestLlama3_1_8BInstruct::test_bfloat16_4gpus[tp2pp2-attn_backend=TRTLLM]` as well? Then, please run `/bot run --stage-list "DGX_H100-4_GPUs-PyTorch-Others-1"` to see if it works.

Reply:
done.